.NET/Microsoft.Recognizers.Text.Sequence/Extractors/BasePhoneNumberExtractor.cs

```diff
@@ -9,7 +9,7 @@ class BasePhoneNumberExtractor : BaseSequenceExtractor
     {
         internal override ImmutableDictionary<Regex, string> Regexes { get; }
 
-        protected sealed override string ExtractType { get; } = Constants.MODEL_PHONE_NUMBER;
+        protected sealed override string ExtractType { get; } = Constants.SYS_PHONE_NUMBER;
```

Comment:
fix phonenumber extractor define

Reply:
I'm not sure this is correct.
For example, for a date, this is the response:
{
  "Text": "dec 2012",
  "Start": 0,
  "End": 7,
  "TypeName": "datetimeV2.daterange",
  "Resolution": {
    "values": [
      {
        "timex": "2012-12",
        "type": "daterange",
        "start": "2012-12-01",
        "end": "2013-01-01"
      }
    ]
  }
}

"bulltin." doesn't show up in response. Same for number.
{
  "Text": "2012",
  "Start": 4,
  "End": 7,
  "TypeName": "number",
  "Resolution": {
    "value": "2012"
  }
}

Reply:
I have checked other "Extractor", and the ExtractorName should be like Constants.SYS_*